.eslintrc.json

```diff
@@ -1,11 +1,4 @@
 {
 	"extends": ["lisk-base"],
-	"rules": {
-		"arrow-parens": "off",
-		"curly": "off",
-		"function-paren-newline": "off",
-		"indent": "off",
-		"no-confusing-arrow": "off",
-		"object-curly-newline": "off"
-	}
+	"plugins": ["mocha"]
```

Comment:
Actually I think this should be in the `test` config - there's an issue in lisk-template to do that too: https://github.com/LiskHQ/lisk-template/issues/89

Reply:
This is where it should be.
It actually complains if it's not here

Reply:
OK got it, the rules are defined on the base config currently so we have no choice.